gradle/libs.versions.toml

```diff
@@ -43,30 +44,32 @@ lombok = "1.18.22"
 mockk = "1.12.2"
 micrometer-prometheus = "1.9.0"
 okhttp3 = "4.9.3"
-postgresql = "42.5.0"
+postgresql = "42.7.3"
 reactor-core = "3.4.14"
 reactor-netty = "1.0.15"
 scala-library = "2.13.9"
 servlet-api = "2.5"
-snakeyaml = "1.32"
-spring-kafka = "2.9.1"
+snakeyaml = "2.2"
+spring-kafka = "2.9.13"
 spring-aws-messaging = "2.2.6.RELEASE"
 aws-java-sdk-s3 = "1.12.342"
-sqlite-jdbc = "3.34.0"
+sqlite-jdbc = "3.43.2.2"
 slf4j = "1.7.35"
 slf4j2 = "2.0.5"
 stellar-wallet-sdk = "1.1.0"
 toml4j = "0.7.2"
+tomcat-embed-core = "9.0.87"
 
 # Plugin versions
 spotless = "6.9.1"
-spring-boot = "2.7.9"
+spring-boot = "2.7.18"
 spring-dependency-management = "1.1.0"
 jacoco = "0.8.10"
 
 [libraries]
 abdera = { module = "org.apache.abdera:abdera-i18n", version.ref = "abdera" }
 apache-commons-lang3 = { module = "org.apache.commons:commons-lang3", version.ref = "apache-commons-lang3" }
+apache-commons-compress = { module = "org.apache.commons:commons-compress", version.ref = "apache-commons-compress" }
```

Comment:
Were this new libraries added anywhere?

Reply:
It is to prevent the vulnerable transitive dependency (`1.21`) from `kafka`
```
org.apache.commons:commons-compress:1.21 -> 1.22
\--- io.confluent:kafka-schema-registry-client:7.0.13
     +--- io.confluent:kafka-json-schema-serializer:7.0.13
     |    \--- compileClasspath
     +--- io.confluent:kafka-json-schema-provider:7.0.13
     |    \--- io.confluent:kafka-json-schema-serializer:7.0.13 (*)
     \--- io.confluent:kafka-schema-serializer:7.0.13
          \--- io.confluent:kafka-json-schema-serializer:7.0.13 (*)
```